src/logic_analyzer.cpp

```diff
@@ -732,6 +735,8 @@ QString LogicAnalyzer::saveToFile()
 	} else {
 		QFile file(fileName);
 		if( !file.open(QIODevice::WriteOnly)) {
+			if(paused)
+				startStop(true);
```

Comment:
as a "hot-fix" this looks fine, but as a general mechanism, i would see about maybe moving the `startStop()` outside of this function;
saving-to-file should not typically do starting/stopping of a state-machine

Reply:
You are right, i will restructure it.